src/person/models.py

```diff
@@ -33,3 +35,6 @@ class Person(models.Model):
     purposes = models.ManyToManyField(Purpose, default=None, blank=True)
     soft_deleted = models.BooleanField(default=False)
     fcm_registration_token = models.TextField(default=None, null=True)
+    is_paused= models.BooleanField(default=False)
+    pause_expiration = models.DateTimeField(default =None, null = True)
```

Comment:
remove spaces around `=` within constructor: `pause_expiration = models.DateTimeField(default=None, null=True)`

Reply:
this would have also been resolved with a pre-commit hook